android-core/src/main/java/com/mparticle/networking/MParticleBaseClientImpl.java

```diff
@@ -185,13 +196,14 @@ protected MPUrl getUrl(Endpoint endpoint, @Nullable String identityPath, boolean
                         .build();
                 return MPUrl.getUrl(uri.toString(), generateDefaultURL(isDefaultDomain, uri, defaultDomain, (pathPrefix + pathPostfix)));
             case AUDIENCE:
-                pathPostfix = SERVICE_VERSION_2 + "/" + mApiKey + "/audience?mpID=" + mConfigManager.getMpid();
-                uri = new Uri.Builder()
-                        .scheme(BuildConfig.SCHEME)
-                        .encodedAuthority(url)
-                        .path(pathPostfix)
-                        .build();
-                return MPUrl.getUrl(uri.toString(), generateDefaultURL(isDefaultDomain, uri, defaultDomain, pathPostfix));
+                    pathPostfix = SERVICE_VERSION_1 + "/" + mApiKey + "/audience";
+                    uri = new Uri.Builder()
+                            .scheme(BuildConfig.SCHEME)
+                            .encodedAuthority(url)
+                            .path(pathPostfix)
+                            .appendQueryParameter("mpID",String.valueOf( mConfigManager.getMpid()))
```

Comment:
nit - not that it matters, but URLs should be all lowercase.
```suggestion
                            .appendQueryParameter("mpid",String.valueOf( mConfigManager.getMpid()))
```